packages/api/lib/writeRecords/write-granules.js

```diff
@@ -515,6 +515,7 @@ const _writeGranuleRecords = async (params) => {
    * @type { { status: string, pgGranule: PostgresGranuleRecord } | undefined }
    */
   let writePgGranuleResult;
+  let limitedUpdateApiGranuleRecord;
 
   log.info('About to write granule record %j to PostgreSQL', postgresGranuleRecord);
   log.info('About to write granule record %j to DynamoDB', apiGranuleRecord);
```

Comment:
At this point `limitedUpdateApiGranuleRecord` is undefined, but I get your point that it might be a bit confusing. What if it said "About to attempt to write granule record %j to DynamoDB" instead?